content/6_Advanced/SCC.mdx

```diff
@@ -171,6 +187,8 @@ public class Main {
 	<Resource source="CPC" title="7 - Graphs 1" url="07_graphs_1" />
 	<Resource source="CP2" title="4.2.9 - Finding SCCs"/>
 	<Resource source="Wikipedia" title="Tarjan's SCC Algorithm" url="https://en.wikipedia.org/wiki/Tarjan%27s_strongly_connected_components_algorithm" />
+	<Resource source="TC" title="Tarjan's Algorithm for SCCs"
+	url="https://www.topcoder.com/thrive/articles/tarjans-algorithm-for-strongly-connected-components"/>
```

Comment:
is this necessary

Reply:
> is this necessary

well ig maybe/maybe not. Firstly I was going to link the dpv but I don't know how to do that. And afterwards I just forgot about it.

Reply:
yeah i'd remove it

Reply:
Done